panther_controller/config/WH04_controller.yaml

```
@@ -0,0 +1,79 @@
+controller_manager:
+  ros__parameters:
+    use_sim_time: false
+    update_rate: 100 # Hz
+
+    joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
+    panther_base_controller:
+      type: diff_drive_controller/DiffDriveController
+    imu_broadcaster:
+      type: imu_sensor_broadcaster/IMUSensorBroadcaster
+
+imu_broadcaster:
+  ros__parameters:
+    sensor_name: "imu"
+    frame_id: "imu_link"
+
+panther_base_controller:
+  ros__parameters:
+    left_wheel_names: ["fl_wheel_joint", "rl_wheel_joint"]
+    right_wheel_names: ["fr_wheel_joint", "rr_wheel_joint"]
+
+    wheel_separation: 0.616
+    wheel_radius: 0.1016
+
+    # For skid drive kinematics it will act as ICR coefficient
+    # Kinematic model with ICR coefficient isn't totally accurate and this coefficient can differ
+    # for various ground types, but checking on our office floor 1.3 looked approximately alright
+    wheel_separation_multiplier: 1.3
+
+    left_wheel_radius_multiplier: 1.0
+    right_wheel_radius_multiplier: 1.0
+
+    # Feedback from motors is published at around 20 Hz
+    publish_rate: 100.0
+    odom_frame_id: odom
+    base_frame_id: base_link
+    pose_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.01]
+    twist_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.01]
+
+    # Whether to use feedback or commands for odometry calculations
+    open_loop: false
+
+    # Update odometry from velocity
+    # in sensor fusion only velocity is used and with this setting it is more accurate
+    position_feedback: false
+    # velocity computation filtering
+    velocity_rolling_window_size: 1
+
+    enable_odom_tf: false
+
+    cmd_vel_timeout: 0.5
+    #publish_limited_velocity: true
+    use_stamped_vel: false
+
+    # Velocity and acceleration limits
+    # Whenever a min_* is unspecified, default to -max_*
+    linear:
+      x:
+        has_velocity_limits: true
+        has_acceleration_limits: true
+        has_jerk_limits: false
+        # top speed is around 2.0m/s
+        max_velocity: 2.0 # m/s
+        # min_velocity -  When unspecified, -max_velocity is used
+        max_acceleration: 2.7 # m/s^2
+        # min_acceleration - When unspecified, -max_acceleration is used.
+        max_jerk: 0.0 # m/s^3
+
+    angular:
+      z:
+        has_velocity_limits: true
+        has_acceleration_limits: true
+        has_jerk_limits: false
+        max_velocity: 4.0 # rad/s
+        # min_velocity -  When unspecified, -max_velocity is used
+        max_acceleration: 5.74 # rad/s^2
+        # min_acceleration - When unspecified, -max_acceleration is used.
```

```suggestion
        # top speed is around 1.2m/s
        max_velocity: 1.1 # m/s
        # min_velocity -  When unspecified, -max_velocity is used
        max_acceleration: 1.5 # m/s^2
        # min_acceleration - When unspecified, -max_acceleration is used.
        max_jerk: 0.0 # m/s^3

    angular:
      z:
        has_velocity_limits: true
        has_acceleration_limits: true
        has_jerk_limits: false
        max_velocity: 2.2 # rad/s
        # min_velocity -  When unspecified, -max_velocity is used
        max_acceleration: 3.2 # rad/s^2
        # min_acceleration - When unspecified, -max_acceleration is used.
```
This values should be the same as for mecanum wheels as radius is the same